pkg/controllers/disruption/drift_test.go

```diff
@@ -434,9 +396,9 @@ var _ = Describe("Drift", func() {
 				Expect(metric.GetGauge().GetValue()).To(BeNumerically("==", 3))
 			}
 
-			// Execute the command in the queue, deleting all nodes
+			// emptiness is not performed in drift, so expect all nodeclaims to exist
```

Comment:
This one too: Do we need to update the naming of our tests/take a look at all the tests in the suite in context to make sure we are still maintaining our testing coverage for the drift disruption?